packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -695,7 +726,6 @@ export class GarbageCollector implements IGarbageCollector {
         this.logIfInactive(
             "Revived",
             toNodePath,
-            this.getCurrentTimestampMs(),
```

Comment:
Why are we no longer using the timestamp value here? Is this because we will just be using the telemetry timestamp?

Reply:
I moved this to the `logIfInactive` method. `logIfInactive` gets the timestamp by calling `this.getCurrentReferenceTimestampMs()` if timestamp is not passed.